src/Views/listings/search.phtml

```diff
@@ -10,22 +10,20 @@ use GroupThr3e\AJExchange\Util\View;
 
 $title = "Search page";
 ?>
-<div class="outerDiv search-grid-container mx-auto p-3 m-5">
+<div class="search-title-div"><h4 class="search-title">Displaying search results for "<?= $params['query'] ?>"</h4></div>
```

Comment:
This div should be horizontally aligned with the rest of the search results. Moving it within the main div should fix this